checker/tests/interning/Raw3.java

```diff
@@ -54,7 +54,7 @@
   // with raw types
   List<@Interned String> foo6() {
     List<String> sl = new ArrayList<>();
-    //:: warning: [unchecked] unchecked conversion
+    //:: warning: [unchecked] unchecked conversion :: error: (return.type.incompatible)
```

Comment:
Because the type of `s1` is `List<@UnknownInterned String>`,  so the type of `(List) s1` should also be `List<@UnknownInterned String>`.  The return type of the method is `List<@Interned String>`, so the returned expression is not a subtype of the return type.